java/ql/lib/change-notes/2023-07-26-flow-out-nonstatic-fields.md

```diff
@@ -0,0 +1,4 @@
+---
+category: minorAnalysis
+---
+* The data flow analysis now allows flow out of fields, even if they are non-static.
```

Comment:
I think this change note is going to be more confusing than helpful, since without a lot of technical background information this can easily imply that we weren't supporting proper field flow before (which is of course wrong).
If we do want a change note (and I would also be fine to skip it), then it should probably be more along the lines of:
```suggestion
* The data flow analysis now supports the declaration of instance fields, rather than their uses, as flow sources using the `FieldValueNode` class.
```

Reply:
I like your suggestion, but probably you're right and the change note isn't necessary after all. I'll remove it.